pathology/multiple_instance_learning/panda_mil_train_evaluate_pytorch_gpu.py

```diff
@@ -0,0 +1,546 @@
+import os
```

Comment:
I feel this file name is not very clear, maybe change it to `train_evaluate.py`?

Thanks.

Reply:
we have a similar name for another pathology tutorial.  I think it should have at least some distinct info in the name.